vignettes/refactor.Rmd

```diff
@@ -174,15 +174,6 @@ factors with the respective encoding using `cfactor`. Further arguments are
 `variable`, `label` and `encoding` referring to the column names in index that 
 contain the respective information.
 
-**Other example strings**
```

Comment:
Would you mind splitting these vignette updates out into a separate pull request please?